### ISSUE TITLE
NVDA+ Chrome: Column with Links of OfficeFabric Detailslist component are not accessible with Keyboard.

### ISSUE THREAD

**Issue by ssthegrt** [nvaccess/nvda#10303]
### Steps to reproduce:
1)Navigate to https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/variablerowheights in Google Chrome
2) Start NVDA as screen reader
3) Try to access each row and the links given using keyboard

### Actual behavior:
1) Columns with more than one link are not accessible using keyboard when NVDA is on.
It starts reading out each single character before moving to next column with link.
![record7](https://user-images.githubusercontent.com/16956669/65860870-cca60300-e388-11e9-9c19-50ba7c601473.gif)

### Expected behavior:
1) All links and rows must be accessible using keyboard.

#### NVDA installed/portable/running from source:
Yes

#### NVDA version:
2019.1.1

#### Windows version:
Windows 10 Enterprise

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by JulienCochuyt** [comment]
> Columns with more than one link are not accessible using keyboard when NVDA is on.

I cannot locate in the big table found on that page any cell with more than one link.
Link are focused fine using table navigation (`control+alt+arrows`, nevertheless, tab navigation seems intercepted or broken in the table.

By the way, while the animated screenshot is interesting, please keep in mind many of us are visually impaired and thus would require the speech output to be also included as a text file in order to be able to consult it.

**Comment by Adriani90** [comment]
@JulienCochuyt I think here is not meant a cell with more than one link, but a column. So in this case I think this is the second column. I am not sure if I understand this issue, I cannot really reproduce it. In browse mode I can access every link by pressing k or by navigating through the table with ctrl+alt+arrow keys. In focus mode I can navigate to every link with arrow keys, although the focus moves in a buggy way. I have to press down arrow and then right arrow to land on the link every time. It seems when pressing down arrow in focus mode the focus always lands on the first column of the row.

**Comment by Adriani90** [comment]
I think this issue occurs mainly in focus mode, NVDA reads the whole row when pressing up and down arrow.
It might also be a web authoring issue here. But I am not sure.

**Comment by Adriani90** [comment]
Yeah it seems this works as expected in focus mode. You can enable it by pressing nvda+space bar. Closing as works for me.
